core/drivers/crypto/caam/hash/caam_hash.c

```diff
@@ -727,11 +674,19 @@ void caam_hash_hmac_copy_state(struct hashctx *dst, struct hashctx *src)
 
 	assert(dst && src);
 
+	if (!dst->initialized)
```

Comment:
Prefer to write this as:
`if (!dst->initialized && caam_hash_hmac_init(dst))`

Reply:
No problem